lib/Pages/ListPage.dart

```diff
@@ -0,0 +1,61 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart';
+import 'dart:convert';
+import 'package:list_maker/API/api_requests.dart';
+import 'package:list_maker/Widgets/add_item_widget.dart';
+import 'package:list_maker/Widgets/items_list_widget.dart';
+
+class NewList extends StatefulWidget {
+  final user;
+  String list_name = "";
+  List uncheckedItems = [];
```

Comment:
What's the type of items in this list?

Reply:
the type of the items is Map. when I specify the type i get an error since the runtime type of the list i get from the server is List<dynamic> so i just left it like that.